kernel/src/debug.rs

```diff
@@ -527,23 +527,45 @@ impl Write for DebugWriterWrapper {
     }
 }
 
-pub fn begin_debug_fmt(args: Arguments) {
+pub fn debug_format(args: Arguments) {
     let writer = unsafe { get_debug_writer() };
 
     let _ = write(writer, args);
-    let _ = writer.write_str("\r\n");
     writer.publish_bytes();
 }
 
-pub fn begin_debug_verbose_fmt(args: Arguments, file_line: &(&'static str, u32)) {
+pub fn debug_formatln(args: Arguments) {
```

Comment:
```suggestion
pub fn debug_println(args: Arguments) {
```

If we're going to rename, I think this makes more sense -- I think I would expect a `formatln` to just write bytes into a buffer; this does the print.

Reply:
Good call. Done.